meson.build

```diff
@@ -26,30 +42,52 @@ if dev_bin_dir == 'undef' or host_bin_dir == 'undef'
   error('dev_bin_dir option not set. Please run meson with a valid binary directory option.')
 endif
 
+# C Arguments to optimize for size
+optimize_size_args = [
+  '-Os', # GCC's General "Optimize for Size" Option
```

Comment:
that's GCC and clang I assume?

Reply:
Yeah, it's both (Clang also has the more severe `-Oz`), so I removed the mention of GCC.